s3/src/main/scala/akka/stream/alpakka/s3/javadsl/S3Client.scala

```diff
@@ -486,8 +486,11 @@ final class S3Client(s3Settings: S3Settings, system: ActorSystem, mat: Materiali
   }
 
   /**
-   * Will return a source of object metadata for a given bucket with optional prefix.
+   * Will return a source of object metadata for a given bucket with optional prefix using version 2 of the LIst Bucket API.
```

Comment:
weird uppercase I

Isn't this the public api where you should note that the behavior depends on the setting?

Reply:
Removed typo and updated both Javadoc and Scaladoc